packages/ui/src/ImageUpload/Image.tsx

```diff
@@ -25,11 +25,13 @@ type Props = {
     style?: { [key: string]: any };
     renderImagePreview?: (props: any) => React.ReactElement<any>;
     round?: boolean;
+    imageContainerStyle?: { [key: string]: any };
```

Comment:
Let's use `React.CSSProperties` instead of `{ [key: string]: any };`.

Example:
```
const style: React.CSSProperties = {
    position: "relative",
    width: 200,
    margin: "0 auto"
};
```

Reply:
Nice!
Should we also change the `style` props as well to follow the same type?

Reply:
Let's do it.